tooling/maven/camel-repackager-maven-plugin/pom.xml

```diff
@@ -0,0 +1,131 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    Licensed to the Apache Software Foundation (ASF) under one or more
+    contributor license agreements.  See the NOTICE file distributed with
+    this work for additional information regarding copyright ownership.
+    The ASF licenses this file to You under the Apache License, Version 2.0
+    (the "License"); you may not use this file except in compliance with
+    the License.  You may obtain a copy of the License at
+
+         http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>org.apache.camel</groupId>
+        <artifactId>maven-plugins</artifactId>
+        <version>4.13.0-SNAPSHOT</version>
+        <relativePath>../pom.xml</relativePath>
+    </parent>
+
+    <artifactId>camel-repackager-maven-plugin</artifactId>
+    <packaging>maven-plugin</packaging>
+
+    <name>Camel :: Maven Plugins :: Repackager</name>
+    <description>Maven plugin to create self-executing JARs using Spring Boot loader tools</description>
+
+    <properties>
+        <firstVersion>4.12.0</firstVersion>
+        <label>tooling</label>
+        <supportLevel>Stable</supportLevel>
+        <spring-boot-version>3.2.7</spring-boot-version>
```

Comment:
Does it have to be such an old SB version. The latest is 3.5.0 which is the version used by Camel Spring Boot